### DISCUSSION TITLE
hover state

### DISCUSSION THREAD

**Discussion by systemes-studio** [beercss/beercss#381] [Ideas]
Hey,

I work with Guillaume Darbonne on www.ROTI.express.
We built our brand new landing page and new app with beercss.
You are doing an amazing job by making material interfaces more accessible for everyone.

I am thinking about some new styles that could improve UX.
We could add a pointer (that already exists for selects) on the hover state of:
1. date pickers where the select mode is forced
2.time pickers where the select mode is forced
3. articles with summaries
4. an entire clickable row in a table
5. inputs for file uploading

This new hover state should also include a border in primary-color (for example) for:
1. date pickers
2. time pickers
3. selects
4. fields
5. text-areas
6. articles with summaries with a border class
7. an entire clickable or non clickable row in a table

This could be a great improvement for visual accessibility.
I can see that you already did a very good job with the focus style!

All the best,

**Comment by laurentpayot** [comment]
Hi @systemes-studio, very cool landing page!

Do your style suggestions comply to the [Material Design specifications](https://m3.material.io/components/date-pickers/specs)?

**Reply by systemes-studio** [reply]
Hi Laurent,
Thanks for your kind words, don't hesitate to sign up on https://app.roti.express to explore the app!
You can check a better example here:
[Material Design ](https://m3.material.io/components/text-fields/specs#41bd8a7f-2760-459c-a245-5547831a1633)
As you can see, there is a hover state with a pointer that turns the field's' border into primary.
This is actually something that is missing in beer css and it would be a great improvement to its UX!
Thank you for your help,
